modules/javafx.graphics/src/main/java/javafx/css/Stylesheet.java

```diff
@@ -133,18 +141,29 @@ public void setOrigin(StyleOrigin origin) {
     /**
      * Constructs a Stylesheet using the given URL as the base URI. The
      * parameter may not be null.
-     * @param url
+     *
+     * @param url the base URI for this stylesheet
      */
     Stylesheet(String url) {
 
         this.url = url;
 
     }
 
+    /**
+     * The rules that are defined in this {@code Stylesheet}
```

Comment:
Same comment as for getOrigin about starting with a verb. Also, the description should end with a period.